README.md

```diff
@@ -58,6 +58,15 @@ icons/
 
 ### Offline Maps
 
+Use [mapbox-style-downloader](https://github.com/digidem/mapbox-style-downloader) to download styles, tiles, glyphs, and sprites for offline use.
+
+To create an asar file from the tiles:
```

Comment:
Actually, this is missing from the docs in mapbox-style-downloader: There is an `--asar` option for the downloader that will do that for you. Can you add that to the downloader docs?